autoload/lsp/hover.vim

```diff
@@ -72,12 +72,14 @@ enddef
 
 # process the 'textDocument/hover' reply from the LSP server
 # Result: Hover | null
-export def HoverReply(lspserver: dict<any>, hoverResult: any): void
+export def HoverReply(silent: bool, lspserver: dict<any>, hoverResult: any): void
```

Comment:
As much as possible, all the functions that deal with the server take the lspserver as the first argument.  Also I prefer the most significant input parameters to be earlier in the order of arguments.  Can you move the "bool" argument to the end?

Reply:
i think it should be, but i had explained, actually i do not know how.
please check: 
`export def DocSymbolReply(fname: string, lspserver: dict<any>, docsymbol: any)`
and
`def DocHighlightReply(bnr: number, lspserver: dict<any>, docHighlightReply: any): void`
etc.
would you mind refactor those all later by you or someone else?